packages/components/src/components/timeline/timeline.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import classNames from 'classnames';
+import Text from '../text';
+
+type TOval = {
+    children: number;
+};
+
+const Oval = ({ children }: TOval) => {
+    return (
+        <div className='dc-timeline__oval'>
+            <Text color='colored-background' size='s' weight='bold' className='dc-timeline__number'>
+                {children}
+            </Text>
+        </div>
+    );
+};
+
+type TItem = React.HTMLAttributes<HTMLDivElement> & { item_title: string | React.ReactNode };
+
+type TTimeline = React.HTMLAttributes<HTMLDivElement> & { disabled_items: number[] };
+
+const Timeline = ({ children, disabled_items, ...props }: React.PropsWithChildren<TTimeline>) => {
+    return (
+        <div {...props}>
+            {Array.isArray(children) &&
+                children.map((child, idx) => (
+                    <div
+                        key={idx}
+                        className={classNames('dc-timeline__flex', {
+                            'dc-timeline__flex--no-border': children.length === idx + 1,
+                            'dc-timeline__flex--disabled': disabled_items && disabled_items.includes(idx + 1),
+                        })}
+                    >
+                        <Oval>{idx + 1}</Oval>
+                        <div className='dc-timeline__container'>
+                            <Text as='h2' color='prominent' size='xs' className='dc-timeline__title'>
+                                {child.props.item_title}
+                            </Text>
```

Comment:
can we change this to {child?.props?.item_title} ?

Reply:
as I see in the code, we pass title everywhere, So I believe this prop should be necessary.
if it is necessary, we should stay without question marks, how do you think?